TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/Cache/ContentCache.php

```diff
@@ -237,11 +243,14 @@ public function getCachedSegment($uncachedCommandCallback, $typoScriptPath, $cac
     public function replaceCachePlaceholders(&$content, $addCacheSegmentMarkersToPlaceholders)
     {
         $cache = $this->cache;
+        $debugger = $this->debugger;
```

Comment:
man könnte der closure die instanz übergeben, das stimmt. Ich habe das eben unobtrusive erweitert, siehe *$cache Variable* sonst müsste ich auch *$that = $this;* definieren und dann der closure $that übergeben mit use